code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -292,4 +292,33 @@ obj/item/weapon/gun/energy/laser/retro
 		if(!power_supply) return 0
 		power_supply.give(100)
 		update_icon()
-		return 1
\ No newline at end of file
+		return 1
+
+
+/obj/item/weapon/gun/energy/megabuster
+	name = "Mega-buster"
+	desc = "An arm-mounted buster toy!"
+	icon_state = "megabuster"
+	item_state = null
+	w_class = 2.0
+	projectile_type = /obj/item/projectile/energy/megabuster

```

Comment:
"/obj/item/projectile/energy/megabuster"
Maybe they added it to text2path now that it just returns the path if it's given a path.
But otherwise it breaks shit.